connectors/katalog/example/asset.yaml

```diff
@@ -3,27 +3,23 @@ kind: Asset
 metadata:
   name: data-csv
 spec:
-  secretRef: 
-    name: data-csv-creds
-  assetDetails:
-    dataFormat: csv
+  vaultPluginPath: vaultPluginPath
+  details:
+    dataFormat:
+      type: csv
     connection:
       type: s3
       s3:
         endpoint: "http://minio:9000"
         bucket: demo
         objectKey: data.csv
-  assetMetadata:
+  resource:
+    name: demoAsset
+    owner: Alice
     geography: us-south
     tags:
-    - finance
-    componentsMetadata:
-      nameOrig: 
+      tag: PII
+    columns:
+      - name: c1
         tags:
```

Comment:
Why do tags appear twice?

Reply:
I suppose it's different tags: tags for the entire asset vs tags for specific columns.. But maybe @rohithdv @ritwikc can elaborate, I was following the schema they defined :)